game/scripts/vscripts/modifiers/player/modifier_player_Asprose.lua

```diff
@@ -0,0 +1,35 @@
+modifier_player_Asprose = class({})
+
+function modifier_player_Asprose:IsPurgable() return false end
+function modifier_player_Asprose:RemoveOnDeath() return false end
+function modifier_player_Asprose:GetTexture() return "player/Asprose" end
```

Comment:
```suggestion
function modifier_player_Asprose:GetTexture() return "player/asprose" end
```
开头使用小写字母。
另外图片需要做成128*128的png 放在指定目录下
参考 https://github.com/windybirth/windy10v10ai/blob/main/game/resource/flash3/images/spellicons/player/76.png